### ISSUE TITLE
timezone option in tztime ignored

### ISSUE THREAD

**Issue by olhotak** [ultrabug/py3status#110]
My config file is the following:
```
order += "tztime local"
order += "tztime toronto"

tztime local {
        format = "%Y-%m-%d %H:%M"
}

tztime toronto {
        format = "%H:%M %Z"
        timezone = "America/Toronto"
}
```

With this config file, `i3status` displays:
`2015-07-23 22:02 | 16:02 EDT`

However, `i3bar` configured with `py3status` displays (in the status bar):
`2015-07-23 22:02 | 22:02`

Notice that:
1. The time is in the wrong timezone (22:02 instead of 16:02).
2. The timezone identifier (EDT) is missing (yet it is supposed to be there due to the `%Z` in the format string).

I would expect `py3status` to match `i3status`.

The JSON output of `py3status` with this configuration file is:
```
{"click_events": true, "version": 1}
[
[{"name":"tztime","full_text":"2015-07-23 22:06"},{"name":"tztime","full_text":"16:06 EDT"}]
,[{"name": "tztime", "full_text": "2015-07-23 22:06"}, {"name": "tztime", "full_text": "22:06 "}]
```

My versions are:
```
py3status version 2.4 (python 2.7.10)
i3status 2.9 (2015-03-22) © 2008-2012 Michael Stapelberg and contributors
i3bar version 4.10.2 (2015-04-16, branch "4.10.2") © 2010-2014 Axel Wagner and contributors
```

**Comment by ultrabug** [comment]
Hi @olhotak 

The problem here is the "%Z" I think and it's a tricky one because the C and python implementation of it is different.

First of all, can you confirm that removing `%Z` fixes the issue ?

**Comment by olhotak** [comment]
Thank you for replying so quickly.

Yes, removing `%Z` fixes the first issue: the timezone is now correct.

**Comment by ultrabug** [comment]
FYI as you can see, I've coded a workaround which allows %z to be used now. This will be available in the 2.7 release :)
